pkg/ccl/sqlccl/csv.go

```diff
@@ -738,14 +739,30 @@ func importJobDescription(
 	return parser.AsStringWithFlags(&stmt, parser.FmtSimpleQualified), nil
 }
 
+const importCSVEnabledSetting = "experimental.importcsv.enabled"
```

Comment:
are we calling this "experimental" or "beta"?

Reply:
experimental. See https://github.com/cockroachdb/cockroach/issues/18608#issuecomment-331277940